packages/common/src/routes.ts

```diff
@@ -0,0 +1,8 @@
+import { Address } from "wagmi";
+import { ChainId } from "./chains";
+
+export const Explorer = {
+  applicationPath(chainId: ChainId, roundId: Address, applicationId: string) {
+    return `/#/round/${chainId}/${roundId.toLowerCase()}/${roundId.toLowerCase()}-${applicationId}`;
```

Comment:
Is the double `roundId` here intentional?

Reply:
intentional! the url structure is like that 🙃